public/common/types/explorer.ts

```diff
@@ -22,9 +22,9 @@ export interface IQueryTab {
   content: React.ReactNode
 }
 
-export interface ILogExplorerProps {
-  http: CoreStart['http']
-}
+// export interface ILogExplorerProps {
```

Comment:
Safe to remove the commented export here?

Reply:
yes, I'll remove.